internal/cmd/app.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"github.com/kairos-io/AuroraBoot/deployer"
+	"github.com/kairos-io/AuroraBoot/internal"
+	sdkTypes "github.com/kairos-io/kairos-sdk/types"
+	"github.com/spectrocloud-labs/herd"
+	"github.com/urfave/cli/v2"
+)
+
+func GetApp(version string) *cli.App {
+	return &cli.App{
+		Name:     "AuroraBoot",
+		Version:  version,
+		Authors:  []*cli.Author{{Name: "Kairos authors", Email: "members@kairos.io"}},
+		Usage:    "auroraboot",
+		Commands: []*cli.Command{&BuildISOCmd},
+		Flags: []cli.Flag{
+			&cli.StringSliceFlag{
+				Name: "set",
+			},
+			&cli.StringFlag{
+				Name: "cloud-config",
+			},
+			&cli.BoolFlag{
+				Name: "debug",
+			},
+		},
+		Description: "Auroraboot is a tool that builds various Kairos artifacts suitable to run Kairos on Vms, bare metal, public cloud or single board computers (SBCs).\nIt also provides functionality like network booting to install Kairos. Read more in the docs: https://kairos.io/docs/reference/auroraboot/",
+		UsageText:   ``,
+		Copyright:   "Kairos authors",
+		Action: func(ctx *cli.Context) error {
+			internal.Log = sdkTypes.NewKairosLogger("aurora", "info", false)
+
+			if ctx.Bool("debug") {
+				internal.Log.SetLevel("debug")
+			}
+			c, r, err := ReadConfig(ctx.Args().First(), ctx.String("cloud-config"), ctx.StringSlice("set"))
+			if err != nil {
+				return err
+			}
+			c.ISO.Name = KairosDefaultArtifactName
+			c.ISO.Label = "COS_LIVE"
```

Comment:
this should not even be a var. We need it to be called COS_LIVE IIRC, so we can just have it as a constant somewhere and just use it when needed hardcoded.

Reply:
moved to a constant

Reply:
nope, removed (see the other comment below)